packages/api/core/src/api/init-scripts/init-link.ts

```diff
@@ -18,7 +18,7 @@ const d = debug('electron-forge:init:link');
  * linked.
  */
 export async function initLink<T>(dir: string, task?: ForgeListrTask<T>) {
-  const shouldLink = process.env.LINK_FORGE_DEPENDENCIES_ON_INIT;
+  const shouldLink = process.env.LINK_FORGE_DEPENDENCIES_ON_INIT || process.env.NODE_ENV === 'test';
```

Comment:
Question: should we just set `process.env.LINK_FORGE_DEPENDENCIES_ON_INIT` to `true` in `.mocharc.js`? Would save us from changing a couple of files

Reply:
Yeah, that sounds good to me.

Reply:
[fd55c77](https://github.com/electron/forge/pull/3319/commits/fd55c770a51fc1f08ae4535b6a8b79690968c7f3)